src/mixins/subscribe-mixin.ts

```diff
@@ -0,0 +1,49 @@
+import { LitElement, Constructor, PropertyValues } from "lit-element";
+import { UnsubscribeFunc } from "home-assistant-js-websocket";
+
+export interface HassSubscribeElement {
+  hassSubscribe(): UnsubscribeFunc[];
+}
+
+/* tslint:disable-next-line */
+export const SubscribeMixin = <T extends LitElement>(
+  superClass: Constructor<T>
+): Constructor<T & HassSubscribeElement> =>
+  // @ts-ignore
+  class extends superClass {
+    private _unsubs?: UnsubscribeFunc[];
```

Comment:
Since this an internal function, can we name it `__unsubs` so we never conflict ? (or use a symbol  as a key that is defined above this mixin)